pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -302,9 +319,7 @@ def _register_ddp_hooks(self) -> None:
 
     def configure_ddp(self):
         self.pre_configure_ddp()
-        self._model = DistributedDataParallel(
-            LightningDistributedModule(self.model), device_ids=self.determine_ddp_device_ids(), **self._ddp_kwargs
-        )
+        self._wrap_model()
         self._register_ddp_hooks()
```

Comment:
don't move _wrap_model into its own function. If we're not fitting, we want to skip *all* of pre_configure_ddp, _wrap_model, and _register_ddp_hooks

Reply:
I see. thanks for the suggestion~ The latest commit has updated based on this comment.